redis/cache.go

```diff
@@ -0,0 +1,17 @@
+package redis
+
+import (
+	"gopkg.in/go-redis/cache.v6"
+	"gopkg.in/redis.v6"
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+// NewStandardRedisCache generates a preconfigured redis cache according to our spec, using msgpack for serialization format.
+func NewStandardRedisCache(options *redis.Options) codec *cache.Codec {
+	client := NewStandardRedisClient(options)
```

Comment:
If you reeeeealy want to squeeze this down, you could inline this in your codec :)